src/Avalonia.Controls/Primitives/IPopupHost.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using Avalonia.Controls.Primitives.PopupPositioning;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Controls.Primitives
+{
+    public interface IPopupHost : IDisposable
+    {
+        void SetContent(IControl control);
```

Comment:
`Content` usually refers to data that can be anything, e.g. `object Content`. If the content is a control, the property is usually named `Child`.